### PR TITLE
Enforce consistent order of import statements

### PR DESCRIPTION
**Pull request by davidmehren**
### Component/Part
linting

### Description
This PR adds a Prettier plugin that enforces a consistent order of `import` statements.
It also fixes all corresponding errors and ensures that format errors can be easily fixed in the future by running Prettier as part of ESLint.

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
  - This PR targets the correct branch: `master` for 1.x & docs, `develop` for 2.x

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-/* SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
  *
- *  SPDX-License-Identifier: CC0-1.0
+ * SPDX-License-Identifier: AGPL-3.0-only
```
Comment (DerMolly) [inline .eslintrc.js:4]: Why did you change this? I though we wanted to put all config files under CC-0?

**Review by DerMolly — APPROVED** [review]

**Review by InnayTool — APPROVED** [review]
